cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -130,7 +132,8 @@ def test_json_responses(self):
         outline_url = reverse_course_url('course_handler', self.course.id)
         chapter = ItemFactory.create(parent_location=self.course.location, category='chapter', display_name="Week 1")
         lesson = ItemFactory.create(parent_location=chapter.location, category='sequential', display_name="Lesson 1")
-        subsection = ItemFactory.create(parent_location=lesson.location, category='vertical', display_name='Subsection 1')
+        subsection = ItemFactory.create(parent_location=lesson.location, category='vertical',
```

Comment:
nit: break long lines like this: https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines